### PR TITLE
Mat 5467 500error

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## MADiE PR

Jira Ticket: [MAT-5467](https://jira.cms.gov/browse/MAT-5467)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

**Comment by jkotanchik-SB** [comment]
What was the cause of the 500?

**Comment by sb-cecilialiu** [comment]
> 

Apache isEqualCollection does not check null.

File: src/test/java/cms/gov/madie/measure/utils/MeasureUtilTest.java [hunk 1]
```diff
@@ -606,6 +607,41 @@ public void testIsCqlLibraryNameChangedReturnsTrueForChange() {
...
+            .description("Just a dumb definition")
+            .build();
+    final Measure original = Measure.builder().supplementalData(List.of(supplementalData1)).build();
+    final Measure changed = Measure.builder().supplementalData(Collections.emptyList()).build();
+    boolean output = measureUtil.isSupplementalDataChanged(changed, original);
+    assertThat(output, is(true));
+  }
+
+  @Test
+  public void testIsSupplementalDataChanged_ReturnsFalseForChangedNonNullSupplementalData() {
```
Comment (jkotanchik-SB) [inline src/test/java/cms/gov/madie/measure/utils/MeasureUtilTest.java]: Please rename method. Test returns true for new SDE.
```suggestion
  public void testIsSupplementalDataChanged_ReturnsTrueForNewNonNullSupplementalData() {
```

File: src/test/java/cms/gov/madie/measure/utils/MeasureUtilTest.java [hunk 2]
```diff
@@ -606,6 +607,41 @@ public void testIsCqlLibraryNameChangedReturnsTrueForChange() {
...
+  @Test
+  public void testIsSupplementalDataChanged_ReturnsFalseForNullSupplementalData() {
+    final Measure original = Measure.builder().supplementalData(Collections.emptyList()).build();
+    final Measure changed = original.toBuilder().supplementalData(Collections.emptyList()).build();
+    boolean output = measureUtil.isSupplementalDataChanged(changed, original);
+    assertThat(output, is(false));
+  }
+
+  @Test
+  public void testIsSupplementalDataChanged_ReturnsFalseForOriginalNonNullSupplementalData() {
```
Comment (jkotanchik-SB) [inline src/test/java/cms/gov/madie/measure/utils/MeasureUtilTest.java]: Please rename. Test returns true for removed SDE.
```suggestion
  public void testIsSupplementalDataChanged_ReturnsTrueForRemovedSupplementalData() {
```

**Review by chubert-sb — APPROVED** [review]

**Pull request merged**